plugins/broker/kafka/context.go

```diff
@@ -2,6 +2,7 @@ package kafka
 
 import (
 	"context"
+	"github.com/Shopify/sarama"
```

Comment:
We format as stdlib imports groups together and then external imports grouped together
 Please move the import